CHANGELOG.MD

```diff
@@ -1,3 +1,8 @@
+## v2.0.0, XX March 2021
```

Comment:
Change to correct date before merging

Reply:
On versioning - is this actually a major version release? The reason I'm asking is because it doesn't require any change from the integrators to adopt it.

Reply:
Have updated to minor version release